docs/core-team-process.md

```diff
@@ -0,0 +1,77 @@
+
+:+1: _Advantages:_
+
+* Makes it less likely that one developer may block the others when their tasks take longer than expected.
+* More asynchronous-friendly.
+* Leaves a trail of documentation for each task.
+
+:-1: _Disadvantages:_
+
+* Decomposing tasks well is challenging.
+* Less communication-friendly; we risk a developer on a long-running task feeling isolated.
+
+### 3. Hand-offs
+
+In this method, each developer (or pair) tackles a single problem in serial during their working hours. When the next developer becomes available, the previous one writes a summary of their efforts and progress in a hand-off, either synchronously and interactively in Slack or asynchronously with a pull request comment. Once the next developer is caught up, they make progress and hand off to the next, and so on.
+
+> Example: developer A logs in during their morning and works for a few hours on the next phase of a feature implementation. They make some progress on the model, but don't progress the controller beyond some stubs and don't get a chance to touch the view at all. When developer B logs in, developer A shares their progress with a conversation in Slack or Zoom until developer B is confident that they understand the problem's current state, at which point developer B begins working and making commits to the feature branch. Developer B implements the view, correcting and adding some methods to the model as needed. Finally, developer C logs in, and developers A and C pair to write the controller methods. They write a comment on the pull request as they wrap up describing the changes that they've made together. Developer B returns the next day, puts the finishing touches on the tests, writes or refines some documentation on the new code, and merges the pull request.
+
+:+1: _Advantages:_
+
+* Maximizes knowledge transfer among participants: everyone gets a chance to work on and become familiar with all the system's layers.
+* Ensures that nobody needs to wait when somebody else is stuck.
+* Handles differences in timezones gracefully.
+
+:-1: _Disadvantages:_
+
+* Overlap times need to be negotiated, either by pair programming or using another method to divvy up work. If we all overlap significantly it functionally decays to one of the other solutions.
+* Hand-offs are high communication touchpoints, but the rest of the time is more isolated.
+
+## Ambient socialization
+
+In addition to these strategies, we can take advantage of other technologies to help us feel connected in an ambient way.
+
+* We all open [Teletype](https://teletype.atom.io/) portals as we work, even when not actively pairing, and share the URL in Slack. We join each other's portals in a window on a separate Atom window and watch each other's progress as a background process.
+* We stream to the world on [Twitch](https://twitch.tv) as we work. We sometimes jump into each other's streams to chat or catch up.
```

Comment:
I love this idea! It can also increase transparency to external contributors, and perhaps make "contributing to oss" look less scary to people who are on the fence/curious about contributing.

Reply:
I'm also [smashwilson](https://www.twitch.tv/smashwilson) on Twitch :wink:

I've never streamed myself _coding_ before though. Oddly I think I'm most nervous about people judging my taste in music in doing that 😜